web/lib/supabase/comments.ts

```diff
@@ -12,3 +12,14 @@ export async function getAllComments(contractId: string, maxCount: number) {
   )
   return comments.map((comment) => comment.data)
 }
+
+export async function getComments(limit: number) {
+  let q = selectJson(db, 'contract_comments')
+  q = q
+    .order('fs_updated_time', {
```

Comment:
I think this is the time that the row got added to supabase. But many comments were added initially with a script.

So you should actually be using data->>createdTime, I think.